cloudinit/netinfo.py

```diff
@@ -598,16 +598,19 @@ def route_pformat():
             tbl_v4 = SimpleTable(fields_v4)
             for (n, r) in enumerate(routes.get("ipv4")):
                 route_id = str(n)
-                tbl_v4.add_row(
-                    [
-                        route_id,
-                        r["destination"],
-                        r["gateway"],
-                        r["genmask"],
-                        r["iface"],
-                        r["flags"],
-                    ]
-                )
+                try:
+                    tbl_v4.add_row(
+                        [
+                            route_id,
+                            r["destination"],
+                            r.get("gateway", "0.0.0.0"),
+                            r["genmask"],
+                            r["iface"],
+                            r["flags"],
+                        ]
+                    )
+                except Exception as e:
```

Comment:
Let's use `KeyError` here instead of `Exception`

Reply:
Fixed